src/test/java/org/fundacionjala/coding/jorge/CamelCaseTest.java

```diff
@@ -0,0 +1,59 @@
+package org.fundacionjala.coding.jorge;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Test Camel Case.
+ */
+public class CamelCaseTest {
+    /**
+     * Test case1.
+     */
+    private static final String TC1 = "TestCase";
+    private static final String TC2 = "CamelCaseMethod";
+    private static final String TC3 = "SayHello";
+    private static final String TC4 = "CamelCaseWord";
+    private static final String TC5 = "";
```

Comment:
Don't create constants.
Please create final local variables inside test method as expectedResult and actualResult.